microk8s-resources/wrappers/run-containerd-with-args

```diff
@@ -21,7 +21,7 @@ fi
 # As the snap refreshes runc changes location, eg moves from
 # /snap/microk8s/123/usr/bin/runc to /snap/microk8s/124/usr/runc.
 # containerd-shims need to look for runc in  /snap/microk8s/current/usr/bin/runc
-SNAP_CURRENT="/snap/microk8s/current"
```

Comment:
Here we put /snap.microk8s/current/ in the path on purpose. 

I am not sure this is safe to change.